README.md

```diff
@@ -181,7 +174,12 @@ For `redis` the command would be:
 ```bash
 pip install -e services/redis
 ```
-If you want to install all services in editable mode you can use the `Makefile` with the following command:
+There are optional dev-dependencies that require `poetry`. Those can be installed with:
+```bash
+poetry install -C <path-to-the-service> --only dev --no-root
+```
```

Comment:
What would be an example of one of these dev dependencies?

Reply:
Everything related to testing and linting, e.g. `pytest`, `black`, etc.